elements/pfe-autocomplete/pfe-autocomplete.ts

```diff
@@ -28,7 +28,7 @@ export class AutocompleteShowEvent extends ComposedEvent {
 
 export class AutocompleteSearchEvent extends ComposedEvent {
   constructor(
-    /** The search query */
+    // The search query
```

Comment:
I changed the documentation syntax so it wouldn't autogenerate entries in the "Events" section of the docs.  It was creating duplicate entries.

Reply:
hmmm 🤔 sounds like that's more a bug in analyzer. It would be a shame to lose the docs on those properties. I'll take a look, maybe we can fix it up

Reply:
oh gotcha!

Reply:
found the problem there that was generating events with names like 'searchValue' and solved it. is that what you meant? I couldn't replicate how describing the fields on the event classes caused double-documentation, in any event it seems to work for me at the moment